mbf_costmap_nav/src/mbf_costmap_nav/costmap_planner_execution.cpp

```diff
@@ -77,12 +75,26 @@ uint32_t CostmapPlannerExecution::makePlan(const geometry_msgs::PoseStamped &sta
                                            double &cost,
                                            std::string &message)
 {
+  // transform the input to the global frame of the costmap, since this is an
+  // "implicit" requirement for most planners
+  // note: costmap_2d::Costmap2DROS::getTransformTolerance might be a good idea,
+  // but it's not part of the class API in ros-kinetic
+  const ros::Duration pat(0.5);
```

Comment:
can we pass directly the values to transformPose?  or at least make frame a reference

Reply:
the getGlobalFrameID returns by value for kinetic and melodic, so we cannot use a ref

Reply:
Ack;  can we rename pat as timeout?